tools/wt_verify.py

```diff
@@ -131,28 +240,27 @@ def main():
 
     try:
         command = construct_command(args)
```

Comment:
let's move this out of the try block.

Reply:
https://github.com/wiredtiger/wiredtiger/pull/10164/commits/052ab3b4c85bedfb931aea4e92262e2fd24a4086